LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-where-parsing.html

```diff
@@ -42,7 +42,8 @@
   assert_valid(true, "{}(:hover, :active)", null, "Pseudo-classes inside");
   assert_valid(true, "{}(div):hover", null, "Pseudo-classes after");
   assert_valid(true, "{}(div)::before", null, "Pseudo-elements after");
-  assert_valid(false, "{}(::before)", null, "Pseudo-elements inside");
+  // Should ask clarification from CSSWG
```

Comment:
Are you planning to land this commentary to WPT repo?

Reply:
Yes? I can remove it otherwise